tests/functional/spec/features/calculated_summary.spec.js

```diff
@@ -239,7 +251,14 @@ describe("Feature: Calculated Summary", () => {
       );
 
       $(CurrencyTotalPlaybackPageSkippedFourth.submit()).click();
-
+      $(UnitTotalPlaybackPage.submit()).click();
+      $(PercentageTotalPlaybackPage.submit()).click();
+      $(NumberTotalPlaybackPage.submit()).click();
+      $(CalculatedSummaryTotalConfirmation.submit()).click();
+      expect(browser.getUrl()).to.contain(SetMinMaxBlockPage.pageName);
+      $(SetMinMaxBlockPage.setMinimum()).setValue(100.0);
```

Comment:
Same as the number spec, we should try to submit and ensure we get an updater error message. Will need to be tested for minimum and maximum.